### PR TITLE
Making const things const.

### PR DESCRIPTION
**Pull request by DanielDionne**
Just making const things const.

### CODE REVIEW

**Comment by TinyTyler** [comment]
:clap: Bless you @DanielDionne

File: cocos2dx/base_nodes/CCAtlasNode.h [hunk 1]
```diff
@@ -102,15 +102,15 @@ class CC_DLL CCAtlasNode : public CCNode, public CCRGBAProtocol, public CCTextur
     */
     virtual void updateAtlasValues();
 
-    virtual void draw(void);
+    virtual void draw(void) override;
```
Comment (bpcjeff) [inline cocos2dx/base_nodes/CCAtlasNode.h]: Did we want to keep both "virtual" and "override" keywords here? Not that it matters that much I don't think.

File: cocos2dx/menu_nodes/CCMenu.h [hunk 2]
```diff
@@ -141,27 +141,27 @@ class CC_DLL CCMenu : public CCLayer, public CCRGBAProtocol
...
     */
-    virtual void onExit();
+    virtual void onExit() override;
 
-    virtual void setOpacityModifyRGB(bool bValue) {CC_UNUSED_PARAM(bValue);}
-    virtual bool isOpacityModifyRGB(void) { return false;}
+    virtual void setOpacityModifyRGB(bool bValue) override {CC_UNUSED_PARAM(bValue);}
+    virtual bool isOpacityModifyRGB(void) const override { return false; }
     
     virtual bool isEnabled() { return m_bEnabled; }
```
Comment (yannycheng) [inline cocos2dx/menu_nodes/CCMenu.h]: should it be const too?

File: cocos2dx/base_nodes/CCAtlasNode.h (hunk 1 above)
Comment (DanielDionne) [inline cocos2dx/base_nodes/CCAtlasNode.h]: I started removing them, but then I felt like it makes it more obvious. It's also more visible than the `override` keyword, which sometimes gets pushed over the edge of the window/monitor.

File: cocos2dx/menu_nodes/CCMenu.h (hunk 2 above)
Comment (DanielDionne) [inline cocos2dx/menu_nodes/CCMenu.h]: Yeah. I didn't do all of them, I went through the ones that permeate through Griffin.

**Comment by DanielDionne** [comment]
@bpcjeff says it LGTHim, merging.

**Pull request merged**